### ISSUE TITLE
Feature request: Add unique filter

### ISSUE THREAD

**Issue by yerke** [01mf02/jaq#113]
Thanks for building jaq. It's very helpful.

Is it possible to add unique filter? https://jqlang.github.io/jq/manual/#unique-unique_by

I think it would be useful to a lot of people.

Thanks!

**Comment by kklingenberg** [comment]
jaq already provides the `unique` and `unique_by` filters, as observable [here](https://github.com/01mf02/jaq/blob/v1.0.0/jaq-std/src/std.jq#L82-L83). Are you perhaps using a previous version of jaq?

**Comment by yerke** [comment]
Oh, interesting. I looked on the list of filters on README, and I think tried it with 1.0, but I could be wrong. Let me try again later.
Sorry for confusion, and thank you for the prompt reply.
